diagnostic.md

```diff
@@ -21,13 +21,17 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
+curl GET http://www.example.com/donuts;
 ```
 
 Write an AJAX request to retrieve a list of **all donuts** on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function(){
+  return $.get({
```

Comment:
This is *technically* correct, but we want you to use the .ajax method, as it is easier to configure.
```js
let getDonuts = function () {
  return $.ajax({
    url: 'http://www.example.com/donuts',
    method: 'GET',
  });
};
```
👍